python/tvm/auto_scheduler/measure.py

```diff
@@ -1132,3 +1249,44 @@ def rpc_runner_run(
         print("")
 
     return results
+
+
+# The map stores special registered buffer for measurement
+#  This can be used for sparse workloads when we cannot use random tensors for measurment.
+global special_buffer_table
+special_buffer_table = {}
```

Comment:
I am against to use global variable. In my personal [test branch](https://github.com/antinucleon/tvm/tree/metal), I use pickle to dump all buffers to `tmpdir/workload_key.pkl` to avoid global variable. In setting with ARM64 Python + Multiprocessing, global variable behavior is not same to what we expect on Linux.

Reply:
Please have another look~ 😄 
I've moved the input data to serialize in the SearchTask. (Even though we may still have to get a global table in the search_task.py)